src/Sylius/Component/Core/Model/Order.php

```diff
@@ -468,14 +468,12 @@ public function setShippingState($state)
     }
 
     /**
-     * Gets the last updated shipment of the order
-     *
-     * @return false|ShipmentInterface
+     * {@inheritdoc}
      */
     public function getLastShipment()
     {
         if ($this->shipments->isEmpty()) {
-            return false;
+            return null;
         }
 
         $last = $this->shipments->first();
```

Comment:
Having current implementation, the "last" shipment is the one that was updated at the latest. On the other hand, the "last" payment is the one that is the last in the payments collection... We should have common and consistent logic here. IMO the "last something" should be 'created at the latest'.